test_runner/regress/test_lfc_prewarm.py

```diff
@@ -70,16 +75,21 @@ def test_lfc_prewarm(neon_simple_env: NeonEnv, query: LfcQueryMethod):
     log.info(f"Inserted {n_records} rows")
 
     http_client = endpoint.http_client()
-    if query is LfcQueryMethod.COMPUTE_CTL:
+    if query == QueryMethod.AUTOPREWARM:
+        sleep(offload_secs + 1)
```

Comment:
Any sleep-based waiting in tests almost certainly causes flakiness. Please, rewrite it into waiting for the LFC content to appear in the remote storage. There is a generic helper in python tests for that -- `wait_until()`, see usage in other tests